tasks/codex/edm.py

```diff
@@ -523,3 +556,59 @@ def transfer_data(context, section=None, refresh=True):
         elif sect == 'user':
             UserDataSync.edm_sync_all(refresh=refresh)
         # TODO Organisations at some point
+
+
+@app_context_task()
+def fix_pending_sightings(context):
+    """
+    Fix config on AssetGroupSightings which are not processed
```

Comment:
would be good to say why this is required.

Reply:
yes, this is a good point.  i will add some comments (in code) for clarity.  basically the short version:  the data _inside_ the AGS.config _also_ needs data-transfer magic as well.  this does that.